gcp_variant_transforms/pipeline_common.py

```diff
@@ -361,5 +370,90 @@ def create_output_table(full_table_id, total_base_pairs, schema_file_path):
   result = os.system(bq_command)
   if result != 0:
     raise ValueError(
-        'Failed to create a bigquery table using "{}" command.'.format(
+        'Failed to create a BigQuery table using "{}" command.'.format(
             bq_command))
+
+
+def delete_table(full_table_id):
+  bq_command = _BQ_DELETE_TABLE_COMMAND.format(FULL_TABLE_ID=full_table_id)
+  return os.system(bq_command)
+
+
+def delete_gcs_files(root_path):
+  gcs_command = _GCS_DELETE_FILES_COMMAND.format(ROOT_PATH=root_path)
+  return os.system(gcs_command)
+
+
+class LoadAvro(object):
```

Comment:
What do you say about moving this piece of code, and at least delete_gcs_files() method into a different libs file (not sure if delete_table should live there, up to your discretion)? pipeline_common is getting rather crowded and I think afaiu, usually either preprocess or bq-to-vcf have to also invoke it (or maybe these methods should be invoked in multiple places - not the case for us).